src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.NativeAot.cs

```diff
@@ -1061,17 +1081,19 @@ private unsafe bool TryGetOrCreateObjectForComInstanceInternal(
                 }
                 else
                 {
-                    NativeObjectWrapper wrapper = NativeObjectWrapper.Create(
+                    NativeObjectWrapper newWrapper = NativeObjectWrapper.Create(
```

Comment:
`_lock` is taken at this spot. Is it a problem?

Reply:
Yes, good catch. I'll make sure to update this to not hold the lock here.